vfs/read_write.go

```diff
@@ -230,28 +230,14 @@ func (fh *RWFileHandle) modified() bool {
 	return true
 }
 
-// close the file handle returning EBADF if it has been
-// closed already.
-//
-// Must be called with fh.mu held
+// FlushWrites flushes any pending writes to cloud storage
 //
-// Note that we leave the file around in the cache on error conditions
-// to give the user a chance to recover it.
-func (fh *RWFileHandle) close() (err error) {
-	defer log.Trace(fh.logPrefix(), "")("err=%v", &err)
-	fh.file.muRW.Lock()
-	defer fh.file.muRW.Unlock()
-
-	if fh.closed {
-		return ECLOSED
+// Must be called with fh.muRW held
+func (fh *RWFileHandle) FlushWrites(closeFile bool) error {
```

Comment:
This should be `flushWrites` so it isn't a public symbol - it can't be safely called outside the package anyway with the locking requirement.